ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -1719,6 +1719,11 @@ void Replica::execute_prepared(bool committed)
         }
       }
     }
+
+    if (global_commit_cb != nullptr)
```

Comment:
Why is this necessary?

Reply:
You want to send the commits for the message that opens the network before it is open as opening will mean f == 1 and we will try to send this message to all the other nodes but not yourself and since this message started when f == 0 it expects that you will not send it to the other nodes